src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1PlacementRequestEmailServiceTest.kt

```diff
@@ -71,21 +74,95 @@ class Cas1PlacementRequestEmailServiceTest {
 
     mockEmailNotificationService.assertEmailRequestCount(1)
 
-    val personalisation = mapOf(
-      "applicationUrl" to "http://frontend/applications/${application.id}",
-      "crn" to TestConstants.CRN,
+    mockEmailNotificationService.assertEmailRequested(
+      CRU_EMAIL,
+      notifyConfig.templates.matchRequestWithdrawn,
+      mapOf(
+        "applicationUrl" to "http://frontend/applications/${application.id}",
+        "crn" to TestConstants.CRN,
+      ),
+    )
+  }
+
+  @Test
+  fun `placementRequestWithdrawn does not send email to applicant if placement request not linked to placement application but no email addresses defined`() {
+    val application = createApplication(
+      applicantEmail = null
+    )
+    val booking = BookingEntityFactory()
+      .withApplication(application)
+      .withDefaultPremises()
+      .produce()
+
+    val placementRequest = createPlacementRequest(
+      application,
+      booking,
+      hasPlacementApplication = false,
+    )
+
+    service.placementRequestWithdrawn(placementRequest)
+
+    mockEmailNotificationService.assertNoEmailsRequested()
+  }
+
+  @Test
+  fun `placementRequestWithdrawn does not send email to applicant if placement request linked to placement application `() {
```

Comment:
again would it be useful to explain how this comes to be? e.g. "... when automatically generated on submission of an application with a known release date"?